assets/component-newsletter.css

```
@@ -44,11 +42,13 @@
 
 .newsletter-form__button {
   width: 4.4rem;
-  margin: 0;
-  right: var(--inputs-border-width);
-  top: 0;
-  height: 100%;
+
   z-index: 2;
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-items: center;
+  border: 1px solid rgba(var(--color-foreground));
```

```suggestion
  border: 1px solid rgba(var(--color-foreground));
  height: 54px;
  width: 54px;
```

The button and form are 54px in height in the design and are meant to be consistent. Make it a square by defining the width the same.